src/usr/share/netcheck/netcheck.py

```diff
@@ -151,7 +151,7 @@ def _connect_and_check_dns(self, network_name):
                           'Internet over %s.' % network_name)
 
         overall_success = False
-        connection_successful = self.network_meta.connect(network_name)
+        connection_successful = self.network_helper.activate_network(network_name)
```

Comment:
While I'm not sold on the new method name (I'll save judgment on that until later), if we keep this method name, the assigned variable name should probably also change.

Reply:
I'm OK with the new method name.